superset/reports/commands/alert.py

```diff
@@ -146,8 +146,13 @@ def _execute_query(self) -> pd.DataFrame:
             limited_rendered_sql = self._report_schedule.database.apply_limit_to_sql(
                 rendered_sql, ALERT_SQL_LIMIT
             )
+            query_username = security_manager.get_user_by_username(
```

Comment:
since `get_df` accepts a `str` and `get_user_by_username` returns a `User` this call is unnecessary.